Exercise_1.java

```diff
@@ -1,21 +1,32 @@
-class BinarySearch { 
+class Exercise_1{

```

Comment:
write time, space complexities
try iterative solution as well because is better as it saves space compared to the recursive solution